ui/ui.go

```diff
@@ -85,6 +85,13 @@ func PrintSelected(name, value string, opts ...Option) error {
 	return nil
 }
 
+// Print if the value (eg. a specific flag value) is empty
```

Comment:
Comments should have the format `// <method> ...`, in this case `// PrintlnIfEmpty ...`